Automattic/wp-calypso
# Tailored flows: refresh the logo cropping tool

## Proposed Changes

(WIP) Add a new design for the cropping icon tool. 
@alshakero, I tried to implement most of the design changes without change much of the component behaviour, It is not 100% how the design is, but I am afraid of doing too many changes on the imageCropping component. Let me know what you think. This is still a Work in Progress.

## Testing Instructions
- Pull this branch and run yarn start
- Navigate to `/setup/intro?flow=link-in-bio or flow=newsletter`
- Go to setup step and check the image upload tool

Related to #67848
Fixes #67848
